slides/slides.md

```diff
@@ -172,6 +176,50 @@ class: dense
 * GHA Covid App
 
 
+<div class="flex flex-col absolute bottom-8 right-6" style="width: 200px;">
+Please install Expo app
+👇 Android or iOS 👇
+<img src="/images/qrcode_expo.dev.png" style="height: 200px; width: 200px; margin: 10px 0;">
+https://expo.dev/client
+</div>
+
+---
+routeAlias: summary
+---
+
+<img src="/images/react-native.svg" style="width: 10%; float: left; margin-right: 20px">
```

Comment:
Do we have these images here?

Reply:
Yup, it reuses the same images as used on the first slide:

<img width="959" alt="image" src="https://github.com/nearform/react-native-workshop/assets/29628323/019e5d89-4fd6-436a-85e4-d973db532e71">

Looks like this:

<img width="958" alt="image" src="https://github.com/nearform/react-native-workshop/assets/29628323/d64b8573-4e56-48dc-853b-06ea61e32031">

and for Expo:

<img width="958" alt="image" src="https://github.com/nearform/react-native-workshop/assets/29628323/9a1e8615-37ce-4715-be3c-27adf2ed06f4">

Reply:
perfect!